mibidata/tiff.py

```diff
@@ -311,6 +312,7 @@ def _page_metadata(page, description):
         'miscalibrated': description.get('mibi.miscalibrated'),
         'check_reg': description.get('mibi.check_reg'),
         'filename': description.get('mibi.filename'),
+        'optional_metadata': description.get('mibi.optional_metadata')
```

Comment:
`'point_name': description.get('mibi.description')
`
on line 301 above can now be:
`'point_name': description.get('mibi.point_name')
`
?

Reply:
Fixed. Also, I'm using the recently introduced terminology: `fov_id` and `fov_name`, instead of `point_name`.